frontend/src/components/Editor.test.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import { mount } from 'enzyme';
+import Editor from './Editor';
+
+/*
+  These tests mimic https://github.com/securingsincity/react-ace/blob/master/tests/src/ace.spec.js
+  to ensure that editor properties (placeholder and value) can be properly
+  tested.
+*/
+
+describe('Editor', () => {
+  it('renders without a placeholder and value', () => {
+    const tree = mount(<Editor />);
+    expect(tree).not.toBeNull();
```

Comment:
maybe add snapshots to these tests?

Reply:
The snapshot would only be valid for ensuring a placeholder is not set. The value is not visible within a snapshot. Perhaps
```javascript
const editor = (tree.instance() as any).editor;
expect(editor.renderer.placeholderNode).toBeNull();
expect(editor.getValue()).toBeNull();
```
can be used within this test?

Reply:
It's fine that the value won't be there, most of the tests, at least at this point, deal with the placeholder

Reply:
Gotcha, should I include snapshots or replace with snapshots?

Reply:
If you can verify that the snapshots capture what you're looking for, that can be sufficient